src/utils/supabase.ts

```diff
@@ -841,3 +845,152 @@ export const getChannel = async (
     message: "Successfully fetched the channel!",
   };
 };
+
+export const getTutors = async (
+  queryData: TutorQuery
+): Promise<SupabaseResponse<TutorSelect[]>> => {
+  const query = supabase.from("tutors").select("*");
+
+  addQueryFilters(query, queryData);
```

Comment:
**question:** what does this do? nothing is going to change

Reply:
I agree that nothing is going to change _if_ it's a basic query. But if I remember correctly, the `queryData` argument passed in can vary which may produce different results. @mellophone can verify if I'm completely wrong.

Reply:
The query object is being modified by the addQueryFilters functions since it's being passed by reference. When the query is awaited, then all of the filters that were added in the addQueryFilters functions are applied.